.gitignore

```diff
@@ -1,3 +1,6 @@
+# IDE configs
+.idea
```

Comment:
individual IDE configs should go in your global gitignore, not in every project you happen to touch.

Reply:
Best advice I've gotten all day.